services/121-service/src/registration/services/registrations-pagination.service.ts

```diff
@@ -370,58 +381,68 @@ export class RegistrationsPaginationService {
     }
   }
 
-  private filterOnRegistrationData(
+  private filterOnRegistrationAttributeData(
     query: PaginateQuery,
     queryBuilder: ScopedQueryBuilder<RegistrationViewEntity>,
-    registrationDataRelations: RegistrationDataInfo[],
-    registrationDataNamesProgram: string[],
+    attributeRelations: RegistrationDataInfo[],
   ): ScopedQueryBuilder<RegistrationViewEntity> {
-    const filterableColumnsRegData = this.createFilterObjects(
-      registrationDataNamesProgram,
-      AllowedFilterOperatorsString,
-    );
-    const parsedFilter = parseFilter(query, filterableColumnsRegData);
-    return this.filterRegistrationDataQb(
+    const filterObjects = this.createFilterObjects(attributeRelations);
+    const parsedFilter = parseFilter(query, filterObjects);
+    return this.filterRegistrationAttributeDataQb({
       queryBuilder,
-      registrationDataRelations,
+      attributeRelations,
       parsedFilter,
-    );
+    });
   }
 
   private createFilterObjects(
-    registrationDataNamesProgram: string[],
-    allowedFilterOperators: FilterOperator[],
+    attributeRelations: RegistrationDataInfo[],
   ): Record<string, FilterOperator[] | true> {
     const filterObject = {};
-    for (const name of registrationDataNamesProgram) {
-      filterObject[name] = allowedFilterOperators;
+    for (const r of attributeRelations) {
+      if (r.type === RegistrationAttributeTypes.numeric) {
+        filterObject[r.name] = AllowedFilterOperatorsNumber;
+      } else {
+        filterObject[r.name] = AllowedFilterOperatorsString;
+      }
```

Comment:
```suggestion
    for (const relation of attributeRelations) {
      if (relation.type === RegistrationAttributeTypes.numeric) {
        filterObject[relation.name] = AllowedFilterOperatorsNumber;
      } else {
        filterObject[relation.name] = AllowedFilterOperatorsString;
      }
```
?
I thought we'd have a ESLint-rule for "one letter variables"...